src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,88 @@
+package lotto.view;
+
+import java.text.NumberFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import lotto.Lotto;
+import lotto.domain.Rank;
+
+public class OutputView {
+  private static final NumberFormat NUMBER_FORMAT = NumberFormat.getInstance(Locale.KOREA);
+
+  public static void printPurchaseResult(List<Lotto> lottos) {
+    System.out.println();
+    System.out.println(lottos.size() + "개를 구매했습니다.");
+
+    for (Lotto lotto : lottos) {
+      printLottoNumbers(lotto);
+    }
+  }
+
+  private static void printLottoNumbers(Lotto lotto) {
+    List<Integer> numbers = lotto.getNumbers();
+    List<Integer> sortedNumbers = sortNumbers(numbers);
+    System.out.println(sortedNumbers);
+  }
+
+  private static List<Integer> sortNumbers(List<Integer> numbers) {
+    List<Integer> sortedNumbers = new ArrayList<>(numbers);
+    Collections.sort(sortedNumbers);
+    return sortedNumbers;
+  }
+
+  public static void printWinningStatistics(Map<Rank, Integer> statistics) {
+    System.out.println();
+    System.out.println("당첨 통계");
+    System.out.println("---");
+
+    printRankStatistics(Rank.FIFTH, statistics);
+    printRankStatistics(Rank.FOURTH, statistics);
+    printRankStatistics(Rank.THIRD, statistics);
+    printRankStatistics(Rank.SECOND, statistics);
+    printRankStatistics(Rank.FIRST, statistics);
+  }
+
+  private static void printRankStatistics(Rank rank, Map<Rank, Integer> statistics) {
+    int count = statistics.get(rank);
+    String message = createRankMessage(rank, count);
+    System.out.println(message);
+  }
+
+  private static String createRankMessage(Rank rank, int count) {
+    String matchDescription = createMatchDescription(rank);
+    String prizeDescription = createPrizeDescription(rank);
+    return String.format("%s (%s) - %d개", matchDescription, prizeDescription, count);
+  }
+
+  private static String createMatchDescription(Rank rank) {
+    if (rank == Rank.FIRST) {
+      return "6개 일치";
+    }
+    if (rank == Rank.SECOND) {
+      return "5개 일치, 보너스 볼 일치";
+    }
+    if (rank == Rank.THIRD) {
+      return "5개 일치";
+    }
+    if (rank == Rank.FOURTH) {
+      return "4개 일치";
+    }
+    return "3개 일치";
+  }
+
+  private static String createPrizeDescription(Rank rank) {
+    int prize = rank.getPrize();
+    return NUMBER_FORMAT.format(prize) + "원";
+  }
+
+  public static void printReturnRate(double returnRate) {
+    if (returnRate == (long) returnRate) {
+      System.out.println("총 수익률은 " + String.format("%.1f", returnRate) + "%입니다.");
+    } else {
+      System.out.println("총 수익률은 " + returnRate + "%입니다.");
+    }
```

Comment:
```suggestion
    if (returnRate == (long) returnRate) {
      System.out.println("총 수익률은 " + String.format("%.1f", returnRate) + "%입니다.");
      return;
    }
    System.out.println("총 수익률은 " + returnRate + "%입니다.");
```
반환을 적용하면 else는 생략할 수 있을 거 같아요.